src/mono/mono/mini/mini-llvm-cpp.cpp

```diff
@@ -382,7 +382,7 @@ mono_llvm_call_args (LLVMValueRef wrapped_calli)
 
 	LLVMValueRef *ret = g_malloc (sizeof (LLVMValueRef) * numOperands);
 
-	for (int i=0; i < numOperands; i++) {
+	for (unsigned int i=0; i < numOperands; i++) {
```

Comment:
‘guint’ or ‘size_t’?

Reply:
`llvm::CallBase::getNumArgOperands` returns `unsigned`.

Reply:
It's too bad that there isn't a more humane way to write `for (typename std::decay<decltype(numOperands)>::type i = 0; i < numOperands; i++)`.

Reply:
“guint” is just a mono abbreviation for same, and “size_t” interoperates losslessly & efficiently in this context, albeit maybe slight overkill.